src/Bridge/Symfony/Routing/Router.php

```diff
@@ -71,7 +71,11 @@ public function getRouteCollection()
     public function match($pathInfo)
     {
         $baseContext = $this->router->getContext();
-        $pathInfo = str_replace($baseContext->getBaseUrl(), '', $pathInfo);
+        $baseURL = $baseContext->getBaseUrl();
+        $p = strpos($pathInfo, $baseURL);
```

Comment:
`$p` is unused, this line should be removed

Reply:
Done

Reply:
done